### PR TITLE
[ES] Update `Client/Beatmap editor/AiMod`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

#10342 split into three separate pull request for better review

### CODE REVIEW

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 1]
```diff
@@ -1,177 +1,173 @@
...
+| Breaktime is not suggested for mania maps. | No se sugiere tiempo de descanso para los mapas de mania. | Mapea la parte donde ocurre el tiempo de descanso. | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania. Esta advertencia se puede ignorar con seguridad si se pretende utilizar el tiempo de descanso en cuestión, o si la sección de la canción no se puede mapear. |
+| <!-- Can't confirm -->Easy/Normal diff contains too many speed changes. | La dificultad Easy/Normal presenta una cantidad frecuente de cambios de velocidad de los sliders, lo cual no se recomienda en los [criterios de clasificación](/wiki/Ranking_criteria). | Reduce el número de cambios de velocidad de los sliders en la dificultad Easy/Normal en cuestión. |  |
+| <!-- Can't confirm -->Kiai is toggled very frequently! | ¡El kiai se activa con mucha frecuencia! | Reduce la cantidad de ocurrencias del kiai en todo el mapa. | También conocidos como *kiai flashes*. Esta advertencia se puede ignorar con seguridad si se pretende el uso del kiai en cuestión. |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:125]:
```suggestion
| Easy/Normal diff contains too many speed changes. | La dificultad Easy/Normal presenta una cantidad frecuente de cambios de velocidad de los sliders, lo cual no se recomienda en los [criterios de clasificación](/wiki/Ranking_criteria). | Reduce el número de cambios de velocidad de los sliders en la dificultad Easy/Normal en cuestión. |  |
| Kiai is toggled very frequently! | ¡El kiai se activa con mucha frecuencia! | Reduce la cantidad de ocurrencias del kiai en todo el mapa. | También conocidos como *kiai flashes*. Esta advertencia se puede ignorar con seguridad si se pretende el uso del kiai en cuestión. |
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 2]
```diff
@@ -1,177 +1,173 @@
...
+| This object is stacked on top of another object. | Dos objetos se apilan uno encima del otro en la misma columna en el tiempo, lo que hace que el mapa sea imposible para conseguir una SS. | Encuentra las teclas apiladas o notas mantenidas y mueve o elimina una de ellas. | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania |
 | More than 6 notes simultaneously is not allowed. | Uno o más patrones en el mapa requieren que el jugador presione activamente más de 6 teclas a la vez. En ciertos casos, los [criterios de clasificación de osu!mania](/wiki/Ranking_criteria/osu!mania) lo desaconsejan, ya que la mayoría de los teclados solo permiten hasta 6 pulsaciones de teclas a la vez debido a un fenómeno conocido como [ghosting](https://nelson-miller.com/what-is-keyboard-ghosting-and-how-do-you-prevent-it/). | Revisa tu mapa para asegurarte de que cada columna tenga como máximo 6 pulsaciones de notas simultáneas. | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:56]:
```suggestion
| More than 6 notes simultaneously is not allowed. | Uno o más patrones del mapa requieren que el jugador pulse activamente más de 6 teclas a la vez. En ciertos casos, los [criterios de clasificación de osu!mania](/wiki/Ranking_criteria/osu!mania) lo desaconsejan, ya que la mayoría de los teclados solo permiten pulsar hasta 6 teclas a la vez debido a un fenómeno conocido como [ghosting](https://nelson-miller.com/what-is-keyboard-ghosting-and-how-do-you-prevent-it/). | Revisa tu beatmap para asegurarte de que cada columna tiene como máximo 6 pulsaciones de nota simultáneas. | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania |
```

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 3]
```diff
@@ -1,177 +1,173 @@
...
 
 #### Design
 
 | Mensaje | Explicación | Solución | Notas |
 | :-- | :-- | :-- | :-- |
-| Your beatmap has no background image. | *Se explica por sí mismo* | Encuentre una imagen adecuada y utilícela como imagen de fondo para la dificultad. | Los vídeos no cuentan como imágenes de fondo porque el usuario no está obligado a descargarlos. |
-| File missing:{0} | A tu mapa le faltan ciertos archivos. | Recupere los archivos que faltan o asegúrese de que el mapa no esté tratando de usar archivos que no existen. |  |
+| Your beatmap has no background image. | Tu beatmap no tiene imagen de fondo. | Encuentra una imagen adecuada y úsala como imagen de fondo para la dificultad. | Los vídeos no cuentan como imágenes de fondo porque el usuario no está obligado a descargarlos. |
+| File missing:{0} | A tu mapa le faltan ciertos archivos. | Recupera los archivos que faltan o asegúrate de que el beatmap no esté tratando de usar archivos que no existen. |  |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:63]:
```suggestion
| File missing:{0} | A tu beatmap le faltan ciertos archivos. | Recupera los archivos que faltan o asegúrate de que el beatmap no esté tratando de usar archivos que no existen. |  |
```

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 4]
```diff
@@ -1,177 +1,173 @@
...
 
 #### Timing
 
 | Mensaje | Explicación | Solución | Notas |
 | :-- | :-- | :-- | :-- |
-| All timing sections have a volume below 5%. | *Se explica por sí mismo* | Configure al menos una sección de timing para que tenga un volumen de sonido de 5 % o más. |  |
+| All timing sections have a volume below 5%. | Todas las secciones de temporización tienen un volumen inferior al 5 %. | Configura al menos una sección de temporización para que tenga un volumen de sonido de 5 % o más. |  |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:69]:
```suggestion
| All timing sections have a volume below 5%. | Todas las secciones de tiempo tienen un volumen inferior al 5 %. | Configura al menos una sección de tiempo para que tenga un volumen de sonido de 5 % o más. |  |
```

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 5]
```diff
@@ -1,177 +1,173 @@
...
 
 #### Meta
 
 | Mensaje | Explicación | Solución | Notas |
 | :-- | :-- | :-- | :-- |
-| Drain time should be over 30 seconds. | La duración de tu mapa de principio a fin, sin incluir los tiempos de descanso, es inferior a 30 segundos. | Intente usar una ruleta para terminar el mapa si su canción lo admite. De lo contrario, su canción puede ser demasiado corta. En tal caso, intente extender su archivo .mp3 para alcanzar la marca de 30 segundos. | AiMod no detecta ruletas que pueden extenderse más allá de la marca de tiempo de drenaje de 30 segundos. |
+| Drain time should be over 30 seconds. | La duración de tu mapa de principio a fin, sin incluir los tiempos de descanso, es inferior a 30 segundos. | Intenta usar un spinner para terminar el mapa si tu canción lo admite. De lo contrario, tu canción puede ser demasiado corta. En tal caso, intenta extender tu archivo .mp3 para alcanzar la marca de 30 segundos. | AiMod no detecta spinners que pueden extenderse más allá de la marca de tiempo de drenaje de 30 segundos. |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:75]:
```suggestion
| Drain time should be over 30 seconds. | La duración de tu mapa de principio a fin, sin incluir los descansos, es inferior a 30 segundos. | Intenta usar un spinner para terminar el mapa si tu canción lo admite. De lo contrario, tu canción puede ser demasiado corta. En tal caso, intenta extender tu archivo .mp3 para alcanzar la marca de 30 segundos. | AiMod no detecta los spinners que pueden sobrepasar los 30 segundos de drenaje. |
```

**Review by agatemosu — COMMENTED** [review]
check 1

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 6]
```diff
@@ -1,177 +1,173 @@
...
+| Audio bitrate is higher than 192 kbps. Consider recompressing to CBR 192 kbps or VBR ~1.0. | El archivo `.mp3` del beatmap está codificado en una [tasa de bits](https://es.wikipedia.org/wiki/Tasa_de_bits) superior a 192 kbps, que supera la asignación de los [criterios de clasificación](/wiki/Ranking_criteria). | Vuelve a codificar el archivo `.mp3` para que su tasa de bits esté entre 128 kbps y 192 kbps (siendo 192 kbps la opción recomendada). | Aunque esta advertencia existe en el juego, AiMod nunca muestra la advertencia debido a limitaciones técnicas. |
+| Audio bitrate is lower than 128 kbps. Consider finding a better quality source. | El archivo .mp3 del beatmap está codificado en una [tasa de bits](https://es.wikipedia.org/wiki/Tasa_de_bits) inferior a 128 kbps, que está por debajo de la asignación de los [criterios de clasificación](/wiki/Ranking_criteria). | Cambia el archivo .mp3 a uno con una tasa de bits más alta. | Aunque esta advertencia existe en el juego, AiMod nunca muestra la advertencia debido a limitaciones técnicas. |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:112]:
```suggestion
| Audio bitrate is higher than 192 kbps. Consider recompressing to CBR 192 kbps or VBR ~1.0. | El archivo `.mp3` del beatmap está codificado en una [tasa de bits](https://es.wikipedia.org/wiki/Tasa_de_bits) superior a 192 kbps, que supera lo permitido en los [criterios de clasificación](/wiki/Ranking_criteria). | Vuelve a codificar el archivo `.mp3` para que su tasa de bits esté entre 128 kbps y 192 kbps (siendo 192 kbps la opción recomendada). | Aunque esta advertencia existe en el juego, AiMod nunca muestra la advertencia debido a limitaciones técnicas. |
| Audio bitrate is lower than 128 kbps. Consider finding a better quality source. | El archivo .mp3 del beatmap está codificado en una [tasa de bits](https://es.wikipedia.org/wiki/Tasa_de_bits) inferior a 128 kbps, que está por debajo de lo permitido en los [criterios de clasificación](/wiki/Ranking_criteria). | Cambia el archivo .mp3 a uno con una tasa de bits más alta. | Aunque esta advertencia existe en el juego, AiMod nunca muestra la advertencia debido a limitaciones técnicas. |
```

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 7]
```diff
@@ -1,177 +1,173 @@
...
+| Kiai time is toggled on for less than 15 seconds. | El Kiai time se activa durante menos de 15 segundos. | Extiende la duración del kiai time a más de 15 segundos. | Esta advertencia se puede ignorar con seguridad si se pretende el uso del kiai en cuestión. |
+| The mp3 you are using is a lot longer than the part which is mapped. Consider cropping it to reduce filesize. | El mp3 que estás utilizando es mucho más largo que la parte mapeada. Considera recortarlo para reducir el tamaño del archivo. | Extiende el beatmap o corta la canción. |  |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:114]:
```suggestion
| The mp3 you are using is a lot longer than the part which is mapped. Consider cropping it to reduce filesize. | El mp3 que estás usando es mucho más largo que la parte mapeada. Considera recortarlo para reducir el tamaño del archivo. | Extiende el beatmap o corta la canción. |  |
```

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 8]
```diff
@@ -1,177 +1,173 @@
...
+| Kiai needs an end time point. | El kiai necesita un punto de finalización. | Agrega un nuevo punto de tiempo heredado para que sirva como punto final del kiai. |  |
+| A preview point for this map is not set. Consider setting one from the Timing menu. | No se ha establecido un punto de vista previa para este mapa. Considera configurar uno desde el menú Timing. | Establece un punto de vista previa para el mapa haciendo clic en `Timing` > `Seleccionar la posición actual como punto de vista previa`. |  |
+| Two timing points exist at the same time! | ¡Existen dos puntos de tiempo al mismo tiempo! | Elimina una de las secciones de temporización en conflicto. |  |
+| {0} out of {1} timing sections have a volume below 5%. | {0} de {1} secciones de temporización tienen un volumen inferior al 5 %. | Establece el volumen de los hitsounds de las secciones de temporización silenciosas en un 5 % o más. |  |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:118]:
```suggestion
| {0} out of {1} timing sections have a volume below 5%. | {0} de {1} secciones de tiempo tienen un volumen inferior al 5 %. | Establece el volumen de los hitsounds de las secciones de tiempo silenciosas en un 5 % o más. |  |
```

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 9]
```diff
@@ -1,177 +1,173 @@
...
+| More than 1/3 of the map is Kiai time. Consider reducing this. | Más de 1/3 del mapa es Kiai time. Considera reducir esto. | Reduce el uso del kiai en todo el mapa. | Esta advertencia se puede ignorar con seguridad si se pretende el uso del kiai en cuestión. |
+| More than 1/2 of the TV Size map is Kiai time. Consider reducing this. | Más de la mitad del mapa TV Size es Kiai time. Considera reducir esto. | Reduce el uso del kiai en todo el mapa. | Esta advertencia se puede ignorar con seguridad si se pretende el uso del kiai en cuestión. |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:120]:
```suggestion
| More than 1/3 of the map is Kiai time. Consider reducing this. | Más de 1/3 del mapa es kiai time. Considera reducir esto. | Reduce el uso del kiai en todo el mapa. | Esta advertencia se puede ignorar con seguridad si se pretende el uso del kiai en cuestión. |
| More than 1/2 of the TV Size map is Kiai time. Consider reducing this. | Más de la mitad del mapa TV Size es kiai time. Considera reducir esto. | Reduce el uso del kiai en todo el mapa. | Esta advertencia se puede ignorar con seguridad si se pretende el uso del kiai en cuestión. |
```

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 10]
```diff
@@ -1,177 +1,173 @@
...
+| Kiai isn't snapped! | El punto de partida del kiai especificado no se ajusta a la línea de tiempo. | Ajusta correctamente el punto de inicio del kiai a su posición deseada en la línea de tiempo. |  |
+| Kiai's end isn't snapped! | El punto final del kiai especificado no se ajusta a la línea de tiempo. | Ajusta correctamente el punto final del kiai a su posición deseada en la línea de tiempo. |  |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:122]:
```suggestion
| Kiai's end isn't snapped! | El punto de finalización del kiai especificado no se ajusta a la línea de tiempo. | Ajusta correctamente el punto de finalización del kiai a su posición deseada en la línea de tiempo. |  |
```

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 11]
```diff
@@ -1,177 +1,173 @@
...
+| Breaktime is not suggested for mania maps. | No se sugiere tiempo de descanso para los mapas de mania. | Mapea la parte donde ocurre el tiempo de descanso. | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania. Esta advertencia se puede ignorar con seguridad si se pretende utilizar el tiempo de descanso en cuestión, o si la sección de la canción no se puede mapear. |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:123]:
```suggestion
| Breaktime is not suggested for mania maps. | No se sugiere tiempo de descanso para los mapas de mania. | Mapea la parte donde ocurre el tiempo de descanso. | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania. Esta advertencia se puede ignorar con seguridad si se pretende usar el tiempo de descanso en cuestión, o si la sección de la canción no se puede mapear. |
```

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 12]
```diff
@@ -1,177 +1,173 @@
...
+| Easy/Normal diff contains too many speed changes. | La dificultad Easy/Normal presenta una cantidad frecuente de cambios de velocidad de los sliders, lo cual no se recomienda en los [criterios de clasificación](/wiki/Ranking_criteria). | Reduce el número de cambios de velocidad de los sliders en la dificultad Easy/Normal en cuestión. |  |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:124]:
```suggestion
| Easy/Normal diff contains too many speed changes. | La dificultad Easy/Normal presenta una cantidad frecuente de cambios de velocidad en los sliders, lo cual no se recomienda en los [criterios de clasificación](/wiki/Ranking_criteria). | Reduce el número de cambios de velocidad en los sliders en la dificultad Easy/Normal en cuestión. |  |
```

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 13]
```diff
@@ -1,177 +1,173 @@
...
+| Romanised title contains unicode. | El campo `Romanised Title` en la pestaña General de [Song Setup](/wiki/Client/Beatmap_editor/Song_setup) contiene [caracteres Unicode no estándar](https://en.wikipedia.org/wiki/List_of_Unicode_characters). | Romaniza el nombre del título de la canción en el campo `Romanised Title` siguiendo las [reglas de estandarización de metadatos de los criterios de clasificación](/wiki/Ranking_criteria#metadatos). |  |
+| Countdown is not allowed in mania mode. | La cuenta regresiva no está permitida en el modo mania. | Desmarca la opción `Enable countdown` en la pestaña Design de [Song Setup](/wiki/Client/Beatmap_editor/Song_setup). | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania |
+| Letterboxing is not allowed in mania mode. | El Letterboxing no está permitido en el modo mania. | Desmarca la opción `Letterbox during breaks` en la pestaña Design de [Song Setup](/wiki/Client/Beatmap_editor/Song_setup). | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:135]:
```suggestion
| Letterboxing is not allowed in mania mode. | El letterboxing no está permitido en el modo mania. | Desmarca la opción `Letterbox during breaks` en la pestaña Design de [Song Setup](/wiki/Client/Beatmap_editor/Song_setup). | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania |
```

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 14]
```diff
@@ -1,177 +1,173 @@
...
+| Source conflicts with {0} diff. | El campo `Source` en la pestaña General de [Song Setup](/wiki/Client/Beatmap_editor/Song_setup) difiere entre una dificultad y otra. | Asegúrate de que el campo `Source` sea consistente en todas las dificultades. |  |
+| Tags conflict with {0} diff. | El campo `Tags` en la pestaña General de [Song Setup](/wiki/Client/Beatmap_editor/Song_setup) difiere entre una dificultad y otra. | Asegúrate de que el campo `Tags` sea consistente en todas las dificultades. |  |
+| Uninherited timing points conflict with {0} diff. | La posición de los puntos de tiempo no heredados ([líneas rojas](/wiki/Client/Beatmap_editor/Timing#puntos-de-tiempo-no-heredados)) difiere entre una dificultad y otra. | Asegúrate de que la posición de todos los puntos de tiempo no heredados sean consistentes en todas las dificultades. |  |
+| Audio file conflicts with {0} diff. | El archivo .mp3 utilizado en el mapa difiere entre una dificultad y otra. | Asegúrate de que todas las dificultades utilicen el mismo archivo .mp3 como base. |  |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:151]:
```suggestion
| Audio file conflicts with {0} diff. | El archivo .mp3 usado en el mapa difiere entre una dificultad y otra. | Asegúrate de que todas las dificultades usen el mismo archivo .mp3 como base. |  |
```

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 15]
```diff
@@ -1,177 +1,173 @@
...
+| Letterbox in breaks conflicts with {0} diff. | La opción `Letterbox during breaks` en la pestaña Design de [Song Setup](/wiki/Client/Beatmap_editor/Song_setup) difiere entre una dificultad y otra. | Asegúrate de que la opción `Letterbox during breaks` esté establecida con la misma configuración en todas las dificultades. |  |
+| Audio preview time conflicts with {0} diff. | El punto de vista previa del audio difiere entre una dificultad y otra. | Establece manualmente el punto de vista previa de cada dificultad haciendo clic en `Timing` > `Seleccionar la posición actual como punto de vista previa` para que todos aterricen en el mismo lugar en todas las dificultades. |  |
+| Beatmap exceeds allowed upload size ({0}kb allowed) | El archivo `.osz` comprimido es demasiado grande. | Elimina los archivos no utilizados de la carpeta del mapa y comprime los activos (imágenes, vídeos, hitsounds, etc.) que tengan un tamaño inflado. | Consulta [este artículo](/wiki/Beatmapping/Beatmap_submission#limitaciones) para obtener más información sobre las limitaciones de envío de osu!. |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:156]:
```suggestion
| Beatmap exceeds allowed upload size ({0}kb allowed) | El archivo `.osz` comprimido es demasiado grande. | Elimina los archivos no usados de la carpeta del mapa y comprime los recursos (imágenes, vídeos, hitsounds, etc.) que tengan un tamaño inflado. | Consulta [este artículo](/wiki/Beatmapping/Beatmap_submission#limitaciones) para obtener más información sobre las limitaciones de envío de osu!. |
```

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 16]
```diff
@@ -1,177 +1,173 @@
...
+| This mapset needs at least 2 osu!mania diffs. | Este conjunto de mapas necesita al menos 2 dificultades de osu!mania. | Crea una segunda dificultad para el conjunto. | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania. Esta advertencia se basa en una regla de los criterios de clasificación arcaica que desde entonces se ha vuelto obsoleta. Por lo tanto, se puede ignorar con seguridad siempre que el mapa siga el [requisito de distribución de los criterios de clasificación actuales](/wiki/Ranking_criteria#beatmap). |
 
 ## Defectos
 
 - **AiMod no puede detectar una gran cantidad de problemas** que se consideran imposibles de clasificar según los [criterios de clasificación](/wiki/Ranking_criteria), como objetos potencialmente mal encajados, hitsounds retrasados, imagen de fondo superior a 2.5 MB, etc.
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:168]:
```suggestion
- **AiMod no puede detectar una gran cantidad de problemas** que se consideran imposibles de clasificar según los [criterios de clasificación](/wiki/Ranking_criteria), como objetos potencialmente mal encajados, hitsounds retrasados, imagen de fondo superior a 2,5 MB, etc.
```

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 17]
```diff
@@ -1,177 +1,173 @@
...
 - **AiMod puede advertirle de problemas que no son**, como que el conjunto de mapas necesita dos dificultades estándar en un mapa de [maratón](/wiki/Beatmap/Marathon) y que el [kiai time](/wiki/Gameplay/Kiai_time) se activa por menos de 15 segundos.
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:169]:
```suggestion
- **AiMod puede advertirte de problemas que no son**, como que el conjunto de mapas necesita dos dificultades estándar en un mapa de [maratón](/wiki/Beatmap/Marathon) y que el [kiai time](/wiki/Gameplay/Kiai_time) se activa por menos de 15 segundos.
```

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 18]
```diff
@@ -1,177 +1,173 @@
...
 
-Es recomendable usar otros programas como [Mapset Verifier](https://github.com/Naxesss/MapsetVerifier/releases) hecho por ::{ flag=SE }:: [Naxess](https://osu.ppy.sh/users/8129817) en conjunto con AiMod si lo que buscas es hacer que tu mapa esté [clasificado](/wiki/Beatmap/Category#ranked). Otros programas están más actualizados con los criterios de clasificación y pueden detectar muchos más problemas que AiMod. No obstante, no ignores completamente a AiMod, ya que algunas reglas en los criterios de clasificación las evalúa dicho programa.
+Es recomendable usar otros programas como [Mapset Verifier](https://github.com/Naxesss/MapsetVerifier/releases) hecho por ::{ flag=SE }:: [Naxess](https://osu.ppy.sh/users/8129817) en conjunto con AiMod si lo que buscas es hacer que tu beatmap esté [clasificado](/wiki/Beatmap/Category#ranked). Otros programas están más actualizados con los criterios de clasificación y pueden detectar muchos más problemas que AiMod. No obstante, no ignores completamente a AiMod, ya que algunas reglas en los criterios de clasificación las evalúa dicho programa.
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:173]:
```suggestion
Es recomendable usar otros programas como [Mapset Verifier](https://github.com/Naxesss/MapsetVerifier/releases) por ::{ flag=SE }:: [Naxess](https://osu.ppy.sh/users/8129817) en conjunto con AiMod si lo que buscas es [clasificar](/wiki/Beatmap/Category#ranked) tu beatmap. Otros programas están más actualizados con los criterios de clasificación y pueden detectar muchos más problemas que AiMod. No obstante, no ignores completamente a AiMod, ya que algunas reglas en los criterios de clasificación las evalúa dicho programa.
```

File: wiki/Client/Beatmap_editor/AiMod/es.md [hunk 19]
```diff
@@ -1,177 +1,173 @@
...
-| <!-- Can't confirm -->Kiai is toggled very frequently! | *Se explica por sí mismo* | Reduzca la cantidad de ocurrencias del kiai en todo el mapa. | También conocidos como *kiai flashes*. Esta advertencia se puede ignorar con seguridad si se pretende el uso del kiai en cuestión. |
+| This beatmap is over 6 minutes long. Consider shortening it if it's not a marathon-style map. | Este beatmap dura más de 6 minutos. Considera acortarlo si no es un mapa estilo maratón. | Agrega más descansos en el beatmap o corta la canción. | Esta advertencia se puede ignorar con seguridad si la longitud del mapa en cuestión es la prevista. |
+| Your beatmap is shorter than 45 seconds. Consider making it longer. | Tu beatmap tiene menos de 45 segundos. Considera hacerlo más largo. | Mapea más partes de la canción. | Esta advertencia se puede ignorar con seguridad si la longitud del mapa en cuestión es la prevista. |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/AiMod/es.md:110]:
```suggestion
| Your beatmap is shorter than 45 seconds. Consider making it longer. | Tu beatmap dura menos de 45 segundos. Considera hacerlo más largo. | Mapea más partes de la canción. | Esta advertencia se puede ignorar con seguridad si la longitud del mapa en cuestión es la prevista. |
```

**Review by agatemosu — APPROVED** [review]

**Review by agatemosu — COMMENTED** [review]
ok no

**Review by agatemosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**